internal/service/credentials/resource_credential_issuer_profile_test.go

```diff
@@ -0,0 +1,199 @@
+		ErrorCheck:               acctest.ErrorCheck(t),
+		Steps: []resource.TestStep{
+			// initial profile
+			initialProfile,
+			{
+				Config:  testAccCredentialIssuerProfile_Full(resourceName, name),
+				Destroy: true,
+			},
+			// update profile
+			updatedProfile,
+			{
+				Config:  testAccCredentialIssuerProfile_Full(resourceName, updatedName),
+				Destroy: true,
+			},
+			// changes
+			initialProfile,
+			updatedProfile,
+			initialProfile,
+			{
+				Config:  testAccCredentialIssuerProfile_Full(resourceName, name),
+				Destroy: true,
+			},
+			{
+				Config:  testAccCredentialIssuerProfile_Full(resourceName, updatedName),
+				Destroy: true,
+			},
+		},
+	})
+}
+
+func TestAccCredentialIssuerProfile_InvalidConfig(t *testing.T) {
+	t.Parallel()
+
+	resourceName := acctest.ResourceNameGen()
+	name := acctest.ResourceNameGen()
+
+	environmentName := acctest.ResourceNameGenEnvironment()
+	licenseID := os.Getenv("PINGONE_LICENSE_ID")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheckEnvironment(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		CheckDestroy:             nil,
+		ErrorCheck:               acctest.ErrorCheck(t),
+		Steps: []resource.TestStep{
+			{
+				Config:      testAccCredentialIssuerProfileInvalidConfig_InvalidName(resourceName, name),
+				ExpectError: regexp.MustCompile("Error: Error when calling `CreateCredentialIssuerProfile`: Validation Error : \\[name must not be empty or blank\\]"),
+			},
+			{
+				Config:      testAccCredentialIssuerProfileInvalidConfig_CredentialServuceNotEnabled(environmentName, licenseID, resourceName, name),
+				ExpectError: regexp.MustCompile("Error: Error when calling `ReadCredentialIssuerProfile`: Issuer not found for environment"),
+			},
+		},
+	})
+}
+
+func testAccCredentialIssuerProfile_Full(resourceName, name string) string {
+	return fmt.Sprintf(`
+	%[1]s
+resource "pingone_credential_issuer_profile" "%[2]s" {
+  environment_id = data.pingone_environment.credentials_test.id
+  name           = "%[3]s"
+}`, acctest.CredentialsSandboxEnvironment(), resourceName, name)
+}
+
+func testAccCredentialIssuerProfileInvalidConfig_InvalidName(resourceName, name string) string {
+	return fmt.Sprintf(`
+	%[1]s
+
+resource "pingone_credential_issuer_profile" "%[3]s" {
+  environment_id = data.pingone_environment.credentials_test.id
+  name           = " "
+
+}`, acctest.CredentialsSandboxEnvironment(), resourceName, name)
+}
+
+func testAccCredentialIssuerProfileInvalidConfig_CredentialServuceNotEnabled(environmentName, licenseID, resourceName, name string) string {
+	return fmt.Sprintf(`
+	%[1]s
+
+resource "pingone_credential_issuer_profile" "%[3]s" {
+  environment_id = pingone_environment.%[2]s.id
+  name           = "%[4]s"
+
+}`, acctest.MinimalSandboxEnvironment(environmentName, licenseID), environmentName, resourceName, name)
```

Comment:
Since we're adding the credentials service to `acctest.MinimalSandboxEnvironment`, you can probbaly explicitly declare your own environment definition without credentials in this test

Reply:
updated per suggestion